content/roadmap/en/dojo-6.0.0-release.md

```diff
@@ -1,12 +1,27 @@
 ---
 title: Dojo 6
-date: Q2 2019
+date: Q3 2019
 released: false
 ---
 
 Features coming in Dojo 6.0.0
 
-- A feature
-- A shiny feature
-- A shinier feature
-- The shiniest feature
+- Update minimum TypeScript support
+- New function-based widget API for authoring widgets
+- New Middleware API for use with function-based widgets
```

Comment:
Worth linking these to the ref guides?

Reply:
Was going to update this to point to the github issue/prs once dojo 6 has been released. Lemme think about linking to the ref guide...